src/main/java/com/example/nasastealphoto/model/Camera.java

```diff
@@ -0,0 +1,52 @@
+package com.example.nasastealphoto.model;
+
+import static jakarta.persistence.CascadeType.PERSIST;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.CreationTimestamp;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "cameras")
+public class Camera {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "camera_id_gen")
+    @SequenceGenerator(name = "camera_id_gen", sequenceName = "cameras_seq", allocationSize = 1)
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false, unique = true)
+    private Long nasaId;
+
+    @CreationTimestamp
+    private LocalDateTime createdAt;
+
+    @ToString.Exclude
+    @JsonBackReference
```

Comment:
@JsonBackReference to handle serialization, but this can lead to issues with bidirectional relationships. It is better to JsonManagedReference. In addition, mixing entities with DTO is not a good practice.

Reply:
There is bidiretional relationship between Camera and Picture that's why in Camera model annotated List<Picture> pictures field '@JsonBackReference' and in Picture model field Camera camera annotated with '@JsonManagedReference'

Reply:
Fixed)